src/faebryk/library/Capacitor.py

```diff
@@ -32,6 +36,24 @@ class TemperatureCoefficient(IntEnum):
         X8R = auto()
         C0G = auto()
 
+    def _TC_param_to_str(self, tc: Parameter, required: bool = False) -> str:
+        if isinstance(tc, Constant):
+            return (
+                tc.value.name
+                if isinstance(tc.value, self.TemperatureCoefficient)
+                else str(tc.value)
+            )
+        elif isinstance(tc, Range):
+            return f"{self._TC_param_to_str(tc.min)} - {self._TC_param_to_str(tc.max)}"
+        elif isinstance(tc, Set):
+            return f"Set({', '.join(map(self._TC_param_to_str, tc.params))})"
+        elif isinstance(tc, TBD):
+            return "TBD" if required else ""
+        elif isinstance(tc, ANY):
+            return "ANY" if required else ""
+        else:
+            return type(tc).__name__
+
```

Comment:
Move to util.py under as_unit:
`def enum_parameter_representation(self, param: Parameter, required: bool = False) -> str`